doc/history.txt

```diff
@@ -55,6 +55,7 @@ we will remove the features in the next major release.
 
 Added/fixed:
 ============
+(*) CTL-904 Fixing one of issues that occurs in multi-thread environment

```

Comment:
I prefer "Allow view models to be created on multiple threads (thread-safe view model factory)"